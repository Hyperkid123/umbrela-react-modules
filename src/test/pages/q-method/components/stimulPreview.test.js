import React from 'react'
import StimulPreview from '../../../../pages/q-method/components/stimulPreview';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Stimul preview component', () => {
  it('should render correclty' , () => {
      const tree = shallow(
        <StimulPreview images={[{name: 'foo', preview: 'foo'}]}/>
      ).dive();
      expect(toJson(tree)).toMatchSnapshot();
  });
});
