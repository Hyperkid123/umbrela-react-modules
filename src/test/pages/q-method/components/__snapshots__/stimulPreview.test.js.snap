// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stimul preview component should render correclty 1`] = `
<div
  className="sc-bdVaJa eVCzwl"
>
  <h2>
    NÃ¡hled
  </h2>
  <div>
    <LazyLoad
      key="foo0"
      offset={0}
      once={false}
      overflow={false}
      resize={false}
      scroll={true}
      unmountIfInvisible={false}
    >
      <ImageThumbNail
        alt="foo"
        src="foo"
      />
    </LazyLoad>
  </div>
</div>
`;
